### ISSUE TITLE
would you mind to upload the weight file?

### ISSUE THREAD

**Issue by Alan2772** [priya-dwivedi/aerial_pedestrian_detection#6]
i wanna do the transfer learning on this repo.
would you mind to upload the trained model weight?
 i train the model follow the instruction but the pedestrian keep below 0.4 and mAP below 0.2.

**Comment by cdshdow** [comment]
> i wanna do the transfer learning on this repo.
> would you mind to upload the trained model weight?
> i train the model follow the instruction but the pedestrian keep below 0.4 and mAP below 0.2.

Have you solved it？I also got a low accuracy.

**Comment by xuxiyang1993** [comment]
I have the same problem. My mAP got lower and lower after each epoch, from .16 to .12

**Comment by mauriziopintus** [comment]
> i wanna do the transfer learning on this repo.
> would you mind to upload the trained model weight?
> i train the model follow the instruction but the pedestrian keep below 0.4 and mAP below 0.2.



> i wanna do the transfer learning on this repo.
> would you mind to upload the trained model weight?
> i train the model follow the instruction but the pedestrian keep below 0.4 and mAP below 0.2.

I also obtained low values, an AP for pedestrians up to 0.39 and a mAP up to 0.19.

**Comment by ahmedselim2017** [comment]
I have same problem too. My mAP is `0.1478` and my pedestirian is `0.2951`
How can we solve that?? @priya-dwivedi

**Comment by priya-dwivedi** [comment]
Hello, trained model is uploaded to:
https://drive.google.com/drive/u/1/folders/1QpE_iRDq1hUzYNBXSBSnmfe6SgTYE3J4

I tried a bunch of things to train the model correctly. These include - passing a config file with smaller anchor points. Config file should also be on github. I also got better results by first training without freezing backbone for a few hours and then using this checkpoint to train longer with freezing backbone.
